src/components/HomePage.js

```diff
@@ -135,49 +116,27 @@ class HomePage extends Component {
-
-                  <p>
-                    I am confident enough to work on any health system app (desktop app,
-                    web app, entreprise app, or mobile app), I am an innovative man,
-                    who loves working with teams on daily basis.
-                  </p>
-
-                  <p>
-                    I graduated from University of Rwanda in College of Medicine
-                    and Health sciences in 2014, and in 2016 I went to Kigali independent
-                    University to study computer sciences, when I grabbed the basics of programming
-                    I dropped out before graduating because I wanted to focus muore on software
-                    engineering in a professiona way. Now I am ready to bring solutions using
-                    programming.
+                   I am a full-stack software engineer with expertise in NodeJS, PostgreSQL, React, Ruby, Ruby on Rails, 
+                   Semantic HTML5, CSS3, Git, Agile methodology, Mobile applications, Technical Team Lead, and mentor.
                   </p>
-
                   <p>
-                    Hard-working, focused, innovative, result-oriented, and growth mindset are
-                    the top four adjectives that best define my aptitude towards work.
+                    I am currently open to full-stack development posts, for global products, startups, and many more.
                   </p>
-
                   <p>
-                    Co-workers see me as a team player who gives his best in sharing ideas,
-                    as well as creating and designing innovative experiences.
+                    Since I started my professional journey in 2014 as a health care provider, I realized that programming 
+                    had the potential to change our world through solving the most challenging, tiresome, and repetitive 
+                    work we do manually.
                   </p>
-
                   <p>
-                    Friends and family members see me as a nerd person, who invest most of his
-                    time for inventing new technology, and who will once move the world to the
-                    extend of changing people lifestyle.
+                    Being a technology enthusiast, I decided to dive deep in programming and developed an android application 
+                    that helps me keep track of my phone logs and SMS which encouraged me that programming is feasible. So I 
+                    joined the Andela stack up to master all of the needed technical and soft skills that professional software 
+                    engineers use in their daily life, and after that, I attended Microverse, an online coding school.
                   </p>
-
                   <p>
-                    Either of them you choose to be, go ahead and get in touch with me!
+                    Now, I can comfortably build solutions based on ideas, help engineering teams to escalate their productivity 
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=6035379)